location/src/main/java/com/yas/location/controller/CountryController.java

```diff
@@ -0,0 +1,88 @@
+package com.yas.location.controller;
+
+import com.yas.location.exception.BadRequestException;
+import com.yas.location.exception.NotFoundException;
+import com.yas.location.model.Country;
+import com.yas.location.service.CountryService;
+import com.yas.location.repository.CountryRepository;
+import com.yas.location.viewmodel.country.CountryPostVm;
+import com.yas.location.viewmodel.country.CountryVm;
+import com.yas.location.viewmodel.error.ErrorVm;
+import com.yas.location.utils.Constants;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import jakarta.validation.Valid;
+import java.util.List;
+
+@RestController
```

Comment:
Consider using `@RequestMapping("/backoffice)`, we have some common path below

Reply:
APIs like getCountries...they can be called from other endpoints like /storefont... so let's stay like this